### PR TITLE
Remove broken links

### PR DESCRIPTION
**Pull request by akshaya-a**

### CODE REVIEW

File: how-to-use-azureml/track-and-monitor-experiments/README.md [hunk 1]
```diff
@@ -12,8 +12,6 @@
 Try out the sample notebooks:
 1. [Use MLflow with Azure Machine Learning for Local Training Run](./train-local/train-local.ipynb)
```
Comment (luigiw) [inline how-to-use-azureml/track-and-monitor-experiments/README.md:13]: you'll need to add /using-mlflow to the first 2 as well

Comment (akshaya-a) [reply how-to-use-azureml/track-and-monitor-experiments/README.md]: done check again please

Comment (luigiw) [reply how-to-use-azureml/track-and-monitor-experiments/README.md]: Looks good to me, doesn't look I have permission to approve though.

**Comment by luigiw** [comment]
I verbally sign off ;)

**Pull request merged**